openlibrary/core/lending.py

```diff
@@ -139,39 +139,39 @@ def get_loan(identifier, user_key=None):
     If user_key is specified, it returns the loan only if that user is
     borrowed that book.
     """
-    d = web.ctx.site.store.get("loan-" + identifier)
-
+    _loan = None
     account = None
     if user_key:
-        from openlibrary.accounts.model import OpenLibraryAccount
         if user_key.startswith('@'):
             account = OpenLibraryAccount.get(link=user_key)
         else:
-            account = OpenLibraryAccount.get(username=user_key.split('/')[-1])
-
-    if d:
-        if (user_key is None or (d['user'] == account.username) or (d['user'] == account.itemname)):
-            loan = Loan(d)
-            if loan.is_expired():
-                loan.delete()
-                return
+            account = OpenLibraryAccount.get(key=user_key)
+
+    d = web.ctx.site.store.get("loan-" + identifier)
```

Comment:
Is this loan information being pulled from the database? Exactly what information is it, if the authoritative loan info is stored in the IA Lists API?